apps/src/aichat/views/assets/staged-files-preview.module.scss

```diff
@@ -56,7 +56,7 @@
   @include fade-in(0.3s);
 }
 
-.preview-pdf {
+.preview-file {
```

Comment:
Above for icon, it looks like we need to use a semantic variable for color. In dark mode, can't see x. I think maybe before this it was only used in `aichat` which is always light mode?

<img width="339" height="181" alt="Screenshot 2025-09-26 at 2 06 13 PM" src="https://github.com/user-attachments/assets/1f40ad88-b490-4f12-a7fa-7bbee5965183" />